setup.cfg

```diff
@@ -1,11 +1,107 @@
+[metadata]
+name = odl
+version = attr: odl.__version__
+description = Operator Discretization Library
+long_description = file: README.md
+long_description_content_type = text/markdown
+author = Jonas Adler, Holger Kohr, and the ODL Contributors
+author_email = odl@math.kth.se
+license = MPL-2.0
+license_file = LICENSE
+url = https://github.com/odlgroup/odl
+download_url = https://github.com/odlgroup/odl/archive/master.zip
+platforms = any
+keywords = research, development, mathematics, prototyping, imaging, tomography
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
```

Comment:
I guess [these](https://pypi.org/classifiers/) are the classifiers. One could add `Education` to the list?

Reply:
Ah good, that's the new link. Thanks! I'll add it as a comment.

I'll also add `Education` -- in the broader sense of science education it seems applicable :wink: